app/models/game.rb

```diff
@@ -67,8 +67,10 @@ def populate_board
    @board
   end
 
-  def is_obstructed(position_1, position_2) # check if proposed move is obstructed
```

Comment:
Hey @denismartens,
I've got some problem with *is_obstructed?* when diagonal movement.
e.g https://gist.github.com/SamuelMartini/522006aa012557db8327
queen.valid_move?([position2]) returns nil when position2 is [2,5] but **true**  when [2,1]
Could you check *is_obstructed?* and *.diagonal_move?* please.